modules/ROOT/examples/syntax/constraints/show.bnf

```diff
@@ -0,0 +1,24 @@
+<show constraints statement> ::= 
+  "SHOW" [ <show constraint type> ] <constraint keywords> [ <show result clause> ]
```

Comment:
more `<_ keywords>` rules that probably should be inlined, and `<show result clause>` that should be defined